setup-gcloud-base/action.yml

```diff
@@ -0,0 +1,29 @@
+name: Setup GCloud
+description: Setup the GCloud CLI
+author: Extenda Retail
+inputs:
+  service-account-key:
+    description: |
+      The service account key which will be used for authentication.
+      The account key should be a base64 encoded JSON key stored as a secret.
+    required: true
+  version:
+    description: Optional gcloud version to install. Defaults to 'latest'.
+    required: false
+    default: latest
+  export-default-credentials:
+    description: |
+      Export the path to the credentials as `GOOGLE_APPLICATION_CREDENTIALS` to be used in subsequent steps.
+      Google Cloud CLIs and APIs will automatically use this environment variable to find credentials.
+    required: false
+    default: 'false'
+  cache:
```

Comment:
Should this be named cache-hit? If this is set to true incorrectly I assume the action fails?

Reply:
`cache-hit` is the output name of the cache@v3 action

Reply:
But I can call it either.

Reply:
>If this is set to true incorrectly I assume the action fails?

Now it's set to `false` by default - so no cache-hit, which means it will install the CLI. In the composite action, it is set to `steps.gcloud-cache.outputs.cache-hit`, so once it finds cache it will become `true` and will only use the cached version.